### DISCUSSION TITLE
Welcome to BioMonTools Discussions!

### DISCUSSION THREAD

**Discussion by leppott** [leppott/BioMonTools#46] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swalther** [comment]
Hi Erik!

I'm an environmental scientist and am up late working my long-term ocean monitoring report chapters (characterizing >50 years of fish and invertebrate trawls, and benthic infauna grabs). I just came across your github contributions and am so excited to see that your monitoring data structures and metrics look similar to mine. Thank you so much for sharing! I've been eager to convert to an automated and reproducible approach to analysis and reporting and your well-documented content gives me hope for the future! Much appreciated!

Here's our most recent receiving water report just in case you're interested. 
[2018-2019 JWPCP Biennial Receiving Water Monitoring Report](https://www.lacsd.org/about-us/what-we-do/ocean-monitoring/reports/)

Take care,
Shelly

**Reply by leppott** [reply]
Thanks.  If you have any suggestions pass them along.  As I get time I add to the package.
